src/actions.cpp

```diff
@@ -479,9 +479,18 @@ void evm_contract::process_tx(const runtime_config& rc, eosio::name miner, const
 
     silkworm::ExecutionProcessor ep{block, engine, state, *found_chain_config->second, gas_params};
 
-    check(tx.max_priority_fee_per_gas == tx.max_fee_per_gas, "max_priority_fee_per_gas must be equal to max_fee_per_gas");
     check(tx.max_fee_per_gas >= _config->get_gas_price(), "gas price is too low");
 
+    if (current_version >= 1) {
```

Comment:
The old behavior `(version==0)` should be to check:
```
check(tx.max_priority_fee_per_gas == tx.max_fee_per_gas, "max_priority_fee_per_gas must be equal to max_fee_per_gas");
check(tx.max_fee_per_gas >= _config->get_gas_price(), "gas price is too low");
```

The behavior for `version>=1` its just to check:
```
auto inclusion_price = min(max_priority_fee_per_gas, max_fee_per_gas - base_fee_per_gas);
eosio::check(inclusion_price >= min_inclusion_price, "inclusion price too low");
```

Reply:
No matter if `min_inclusion_price > 0` or not since `inclusion_price` can't go negative